backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/backup/HybridBackupCommand.java

```diff
@@ -237,6 +237,31 @@ private void auditLog(AuditLogType type) {
         auditLogDirector.log(this, type);
     }
 
+    @Override
+    public AuditLogType getAuditLogTypeValue() {
+        VmBackup vmBackup = getParameters().getVmBackup();
+        addCustomValue("VmName", getVm().getName());
+        addCustomValue("backupId", vmBackup.getId().toString());
+        switch (getActionState()) {
+            case EXECUTE:
+                return AuditLogType.VM_BACKUP_STARTED;
+            case END_FAILURE:
+                if (vmBackup.getPhase() == VmBackupPhase.SUCCEEDED) {
+                    return AuditLogType.VM_BACKUP_SUCCEEDED;
+                }
+
+                return AuditLogType.VM_BACKUP_FAILED;
+            case END_SUCCESS:
+                if (!getSucceeded()) {
+                    return AuditLogType.VM_BACKUP_FAILED;
+                }
+                if (vmBackup.getPhase() == VmBackupPhase.SUCCEEDED) {
+                    return AuditLogType.VM_BACKUP_SUCCEEDED;
+                }
```

Comment:
can we get to line 256 or fallback to 262 here consider that we always call `updateVmBackupPhase(VmBackupPhase.SUCCEEDED);` and `setSucceeded(true);` in endSuccessfully?